internal/cmd/beta/security-group/describe/describe.go

```diff
@@ -0,0 +1,155 @@
+package describe
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/goccy/go-yaml"
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/iaas/client"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/tables"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/iaas"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	SecurityGroupId string
+}
+
+const groupIdArg = "GROUP_ID"
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "describe",
+		Short: "Describes security groups",
+		Long:  "Describes security groups by its internal ID.",
+		Args:  args.SingleArg(groupIdArg, utils.ValidateUUID),
+		Example: examples.Build(
+			examples.NewExample(`Describe an existing group`, `$ stackit beta security-group describe 9e9c44fe-eb9a-4d45-bf08-365e961845d1`),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(p, cmd, args)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p)
+			if err != nil {
+				return err
+			}
+
+			// Call API
+			request := buildRequest(ctx, model, apiClient)
+
+			group, err := request.Execute()
+			if err != nil {
+				return fmt.Errorf("get security group: %w", err)
+			}
+			p.Info("Described security group %q for %q\n", model.SecurityGroupId, model.ProjectId)
```

Comment:
This info log is also not necessary here, the `describe` command already outputs the requested information below